src/unet.py

```diff
@@ -1,76 +1,79 @@
 import tensorflow as tf
-from tensorflow.keras.layers import concatenate 
+from tensorflow.keras.layers import concatenate, Dropout, BatchNormalization, Conv2D, Conv2DTranspose
 
 
 def unet(features, labels, mode) :
     print("called")
-    # Input Layer 
-    input_layer = tf.reshape(features['mix'],[-1,512,128,1])
-    
+    # Input Layer
+    input_shape = (-1,512,128,1)
+    input_layer = tf.reshape(features['mix'],input_shape)
+
     # # Convolutional Layer 1
-    # conv1 = tf.keras.layers.batch_normalization(tf.keras.layers.conv2d(inputs = input_layer, filters=16, kernel_size=[5,5], 
-    #                                                    strides=[2,2], padding="same", activation=tf.nn.leaky_relu))
-    
-    # # Convolutional Layer 2 
-    # conv2 = tf.layers.batch_normalization(tf.layers.conv2d(inputs = conv1, filters = 32, kernel_size = [5,5], 
-    #                                                    strides = [2,2], padding="same", activation = tf.nn.leaky_relu))
+    conv1 = BatchNormalization()(tf.keras.layers.Conv2D(filters=16, kernel_size=[5,5],
```

Comment:
Change this to just `Conv2D(...)` rather than `tf.keras.layers.Conv2D(...)` to be consistent with the other layers.

Reply:
completed